.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: generic
+services: docker
+
+script:
+  - docker build -t rultor-python -f Dockerfile .
+  - docker images
+  # Test building and installing coala
+  - >
+    docker run --rm rultor-python sh -c
+    "git clone https://github.com/coala/coala &&
+     cd coala &&
+     pip3 install -r requirements.txt &&
+     python3 setup.py bdist_wheel &&
+     pip3 install ./dist/coala-*.whl &&
+     cd .. &&
+     git clone https://github.com/coala/coala-bears &&
+     cd coala-bears &&
+     pip3 install -r requirements.txt &&
+     python3 setup.py bdist_wheel &&
+     pip3 install $(ls ./dist/*.whl)\"[alldeps]\" &&
```

Comment:
maybe try printing out `$(ls ./dist/*.whl)` and removing `\"[alldeps]\"` -- the latter is part of a conversion which hasnt happened yet - you may need to manually install `language_check` , as that is the main difference.